### PR TITLE
Swap out the test data set for one that will produce sourmash gather results and has long read data

### PR DESCRIPTION
**Pull request by taylorreiter**
This PR introduces a new test data set: https://github.com/Arcadia-Science/test-datasets/blob/main/seqqc/samplesheet_test.csv

For now, it only swaps out the small test data set. I might swap the large one later, but I want to think about it a bit more before I do that.

This PR also fixes a small bug I found that caused the gather results to not be passed to the pubDir, which caused the gather results not to be included in the multiqc html. i didn't catch it with the old test data set because that data set didn't produce any gather results, so it was normal for it to be missing.

Lastly, I also removed an annoying exit from the samplesheet check file. The lines inappropriately caused nextflow to fail saying that the samplesheet didn't have a header, but it certainly had a header. This has happened to me on three separate occasions so I felt like it was making the code unnecessarily brittle.

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/Arcadia-Science/seqqc/tree/master/.github/CONTRIBUTING.md)
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bin/check_samplesheet.py [hunk 1]
```diff
@@ -158,9 +158,6 @@ def sniff_format(handle):
     peek = read_head(handle)
     handle.seek(0)
     sniffer = csv.Sniffer()
-    if not sniffer.has_header(peek):
```
Comment (mertcelebi) [inline bin/check_samplesheet.py:161]: hmm why?

**Review by mertcelebi — APPROVED** [review]

File: bin/check_samplesheet.py (hunk 1 above)
Comment (taylorreiter) [reply bin/check_samplesheet.py:161]: I have literally no idea. the first time it happened i tried rewriting in vim, opening and re-saving in excel, reading in and writing out in R, re-writing from scratch in excel...so I don't know what it failed.

Comment (taylorreiter) [reply bin/check_samplesheet.py:161]: This just in! @elizabethmcd got this error on an nf-core MAG sample sheet so i feel like this is a script problem and not a me problem.

Comment (elizabethmcd) [reply bin/check_samplesheet.py:161]: yes basically any nf-core MAG samplesheet I try without this commented out it says it doesn't have a header, even though it does, and when I comment this out things go fine

Comment (taylorreiter) [reply bin/check_samplesheet.py:161]: this is what the file looks like with it's special characters...is it missing an `\r` at the end of rows or something? does anyone spot anything?
```
bb =open(filename, "rb")
cc = bb.read()
cc
```
```
b'sample,fastq_1,fastq_2\nSRR8528939,https://github.com/Arcadia-Science/test-datasets/raw/main/seqqc/SRR8528939_10k.fq.gz,\nSRR11550236,https://github.com/Arcadia-Science/test-datasets/raw/main/seqqc/SRR11550236_1_1k.fq.gz,\nSRR3927310,https://github.com/Arcadia-Science/test-datasets/raw/main/seqqc/SRR3927310_1_10k.fq.gz,https://github.com/Arcadia-Science/test-datasets/raw/main/seqqc/SRR3927310_2_10k.fq.gz\n'
```

Comment (taylorreiter) [reply bin/check_samplesheet.py:161]: Looks the same to me as one that doesn't produce this error
```
b'sample,fastq_1,fastq_2\ncomm_1,https://github.com/Arcadia-Science/test-datasets/raw/main/cheese-illumina-metagenomes/comm_1_subsampled_1.fq.gz,https://github.com/Arcadia-Science/test-datasets/raw/main/cheese-illumina-metagenomes/comm_1_subsampled_2.fq.gz\nvir_1,https://github.com/Arcadia-Science/test-datasets/raw/main/cheese-illumina-metagenomes/vir_1_subsampled_1.fq.gz,https://github.com/Arcadia-Science/test-datasets/raw/main/cheese-illumina-metagenomes/vir_1_subsampled_2.fq.gz\n'
```

Comment (taylorreiter) [reply bin/check_samplesheet.py:161]: (im going to merge this for now)

**Pull request merged**